core/org.wso2.carbon.user.core/src/main/java/org/wso2/carbon/user/core/ldap/ActiveDirectoryUserStoreManager.java

```diff
@@ -729,6 +729,8 @@ private String escapeSpecialCharactersForFilter(String dnPartial){
                     case '\\':
                         sb.append("\\5c");
                         break;
+                  /*right now * is used as an IS special character so
```

Comment:
Use proper natural language conventions for logging. For eg: The first letter of the comment could have been made capital.